Mentoring-Program-Guide.md

```diff
@@ -341,3 +341,12 @@ A few things that you might want to do are:
 - Don't forget using [GSoC] or [Outreachy].
 
 - Don't forget using [RFC] or [WIP], if it's relevant.
+
+# Conclusion
+
+Hope you got an idea about how things usually go during the internship.
+As a reminder, don't be surprised if your mentors ask for things that
+are a bit different than what is described here. Typically your mentors
+will expect that you become more autonomous.
+
+Wishing you luck during the internship!
```

Comment:
Same thing about "internship".

BTW maybe we should already work on scripts detecting and possibly replacing some terms.

Reply:
> BTW maybe we should already work on scripts detecting and possibly replacing some terms.

Yeah. If we have a glossary, this is definitely a good next step.